mappings/net/minecraft/client/render/block/entity/BlockEntityRenderer.mapping

```diff
@@ -1,6 +1,16 @@
 CLASS net/minecraft/class_827 net/minecraft/client/render/block/entity/BlockEntityRenderer
 	FIELD field_4367 renderManager Lnet/minecraft/class_824;
 	FIELD field_4368 DESTROY_STAGE_TEXTURES Ljava/util/List;
+	METHOD method_22747 render (Lnet/minecraft/class_2586;DDDFILnet/minecraft/class_287;Lnet/minecraft/class_1921;Lnet/minecraft/class_2338;)V
+		ARG 1 blockEntity
+		ARG 2 xOffset
+		ARG 4 yOffset
+		ARG 6 zOffset
+		ARG 8 tickDelta
+		ARG 9 blockBreakStage
```

Comment:
Imo the `block` prefixes in the parameters here aren't necessary given the context, but it's just a small nitpick :P

Reply:
breakingStage? I'd still probably keep blockEntity